src/main/scala/lenthall/config/ConfigValidationException.scala

```diff
@@ -0,0 +1,12 @@
+package lenthall.config
+
+import lenthall.exception.MessageAggregation
+
+import scalaz.NonEmptyList
+
+case class ConfigValidationException(context: String, errors: NonEmptyList[String]) extends RuntimeException with MessageAggregation {
+  def this(context: String, errorMessage: String) = this(context, NonEmptyList(errorMessage))
+
+  override val errorMessages = errors.list
+  def message = s"Invalid $context configuration"
```

Comment:
this "message" seems more like the context than the Throwable sense of message below